kiwi/shareddata.h

```diff
@@ -16,22 +16,25 @@ class SharedData
 public:
     SharedData() : m_refcount(0) {}
 
-    SharedData(const SharedData &other) : m_refcount(0) {}
+    SharedData(const SharedData &other) = delete;
+
+    SharedData(SharedData&& other) = delete;
 
     int m_refcount;
 
-private:
-    SharedData &operator=(const SharedData &other);
+    SharedData &operator=(const SharedData &other) = delete;
+    
+    SharedData &operator=(SharedData&& other) = delete;
```

Comment:
```suggestion
    SharedData &operator=(const SharedData &other) = delete;
    SharedData &operator=(SharedData&& other) = delete;
```

No need for all these blank lines.

Reply:
This was for respecting code convention